### PR TITLE
Add Random Episode Play Button to TV Season page

### PR DESCRIPTION
**Pull request by 1hitsong**
**Changes**
I added a button to the TV Season page that selects a random episode and plays it. 

To accommodate the button, and planning for future buttons such ask shuffle play, I had to adjust the layout/design of the TV Season page slightly to create room for the action buttons. 

One important note, I did hide the codec information at the bottom of each episode item to help reduce the height. Does anyone see this as required information that needs to be worked back in? It's not info that's shown on the season web client screen.

### CODE REVIEW

**Review by candry7731 — APPROVED** [review]
This works great! 

I did notice the episode count blue circle is really small... not sure if this had anything to do with this pr.

**Comment by 1hitsong** [comment]
I believe the blue circle is added by the server when requesting the poster image. It's unrelated to this PR.

File: components/tvshows/TVEpisodes.xml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 <?xml version="1.0" encoding="utf-8" ?>
 <component name="TVEpisodes" extends="JFGroup">
   <children>
+    <Poster id="seasonPoster" width="300" height="450" translation="[95,175]" />
+    <JFButton id="Random" text="Play Random" translation="[90, 640]"></JFButton>
```
Comment (jimdogx) [inline components/tvshows/TVEpisodes.xml]:
```suggestion
    <JFButton id="Random" text="Play Random" translation="[90, 640]" visible="false"></JFButton>
```

File: components/tvshows/TVEpisodes.brs [hunk 2]
```diff
@@ -10,12 +15,34 @@ sub setSeasonLoading()
 end sub
 
 sub updateSeason()
+    imgParams = { "maxHeight": 450, "maxWidth": 300 }
+    m.poster.uri = ImageURL(m.top.seasonData.Id, "Primary", imgParams)
```
Comment (jimdogx) [inline components/tvshows/TVEpisodes.brs:19]:
```suggestion
    m.poster.uri = ImageURL(m.top.seasonData.Id, "Primary", imgParams)
    m.Random.visible = true
```

**Comment by jimdogx** [comment]
The code information was a feature request.  It also lets you see which episodes have different codecs to choose from (commentary tracks for example).

File: components/tvshows/TVEpisodes.xml (hunk 1 above)
Comment (jimdogx) [reply components/tvshows/TVEpisodes.xml]: On my setup, the Play Random is showing for quite some time before the screen is actually loaded.  I'd suggest making it invisible until everything is done loading.

**Comment by 1hitsong** [comment]
Cool. I didn't remove, only hid it. I'll do a little design rework to work it in nicely.

**Comment by 1hitsong** [comment]
@jimdogx Added back

**Review by jimdogx — APPROVED** [review]

**Pull request merged**